geotrek/api/v2/views/tourism.py

```diff
@@ -98,3 +99,14 @@ def get_queryset(self):
                               ) \
             .annotate(geom_transformed=Transform(F('geom'), settings.API_SRID)) \
             .order_by('name')  # Required for reliable pagination
+
+
+class TouristicEventPlaceViewSet(api_viewsets.GeotrekGeometricViewset):
+    filter_backends = api_viewsets.GeotrekGeometricViewset.filter_backends + (
+        api_filters.NearbyContentFilter,
+        api_filters.UpdateOrCreateDateFilter,
+    )
+    serializer_class = api_serializers.TouristicEventPlaceSerializer
+
+    def get_queryset(self):
+        return tourism_models.TouristicEventPlace.objects.exclude(touristicevents__isnull=True).annotate(geom_transformed=Transform('geom', settings.API_SRID)).order_by('name')
```

Comment:
why excluding those with no touristic event ? I think API should return all places, and use filter to get those with event linked

Reply:
It's meant to be like Categories and Types, that are returned only if there is published content for them

Reply:
but is there a reason to publish places ?

Reply:
We did it this way, as I think it is the same on all API routes, even InformationDesk that is similar to EventPlaces.
But yes, it can be discussed.